tests/syntax/test_modular.py

```diff
@@ -809,6 +810,77 @@ def test_override_behavior():
     assert tuple(actions) == (1, -1, -2, 2)
 
 
+def test_override_none_behavior():
+    scenario = compileScenic(
+        """
+        scenario Main():
+            setup:
+                ego = new Object
+            compose:
+                wait
+                do Sub() for 2 steps
+                wait
+        scenario Sub():
+            setup:
+                override ego with behavior Bar
+        behavior Bar():
+            x = -1
+            while True:
+                take x
+                x -= 1
+        """,
+        scenario="Main",
+    )
+    actions = sampleEgoActions(scenario, maxSteps=4)
+    assert tuple(actions) == (None, -1, -2, None)
+
+
+def test_override_leakage():
+    scenario = compileScenic(
+        """
+        scenario Main():
+            setup:
+                ego = new Object with prop 1
+            compose:
+                do Sub1()
+        scenario Sub1():
+            setup:
+                override ego with prop 2
+                override ego with behavior Bar
```

Comment:
```suggestion
                override ego with prop 2, with behavior Bar
```
This is clearer, I think -- otherwise someone might think the test is about some bad interaction between multiple `override` statements.